File: java/Day06/Demo01/src/com/sunbeam/Person.java
package com.sunbeam;

import java.util.Scanner;

public class Person {
	String name;

	public void accept(Scanner sc) {
		System.out.print("Enter the name - ");
		name = sc.next();
	}

	public void display() {
		System.out.println("Name - " + name);
	}
}

File: cpp/Day11/attendancesystem/person.cpp
#include "person.h"

Person::Person()
{
    cout << "Person()" << endl;
}
Person::Person(int id, string name)
{
    cout << "Person(int name)" << endl;
}
void Person::accept()
{
    cout << "Person::accept()" << endl;
}
void Person::display()
{
    cout << "Person::display()" << endl;
}